episodes/collaborating-team.md

```diff
@@ -0,0 +1,256 @@
+        and then how to mark as it as ready for review.
+1. \(S) Request a review from the Pull Request Reviewer.
+1. \(R) Open the new Pull Request in GitHub.
+1. \(R) Explore the diff of the PR,
+        by opening the _Files changed_ tab and talking through the display.
+1. \(R) Comment on a line in the diff.
+1. \(R) Suggest a change to the diff,
+        using the "Add suggestion" button or three backticks marked `suggestion`.
+1. \(R) Complete your review by adding some summary text and selecting "Request changes".
+1. \(R) Return to _Conversation_ tab,
+        to show how the PR thread looks after the review has been submitted.
+1. \(S) Navigate to PR and the _Files changed_ tab.
+1. \(S) Respond to Reviewer's comment(s), accept their suggested changes
+        (draw attention to the option to batch multiple suggestions into a single commit).
+1. \(S) Return to the _Conversation_ tab and request a new review from the Reviewer.
+1. \(R) Look again at the PR diff (_Files changed_)
+        and ensure that requested changes have been made.
+1. \(R) Approve the PR.
+1. \(R) Thank the Submitter for their contribution.
+1. \(R) Merge the PR.
+1. \(R) Navigate to issue listing to confirm that the relevant issue has been
+        automatically closed.
+
+
+:::::: challenge
+
+### Practice with Pull Requests (15 minutes)
+
+In breakout groups, assign yourselves the role of Submitter (S) and Reviewer (R),
+and repeat the process described above by completing the steps below.
+If there are any issues open on the repository that can be solved very quickly,
+feel free to address those with this exercise.
+Otherwise, the Submitter can open a Pull Request to make simple changes:
+typo fixes, or some completely arbitrary changes to a file.
+If the changes made for the exercise are not improvements to the lesson,
+the Reviewer should follow all the steps but close the PR at the end instead of merging it.
+
+1. \(S) Edit a file and commit the changes to a new branch.
+2. \(S) Open a new Pull Request from this new branch to `main`.
+        If your PR will fix an open issue on your repository,
+        include "Fixes #X" at the start of the PR description,
+        where _X_ is the issue number.
+3. \(S) Request a review from the Reviewer.
+4. \(R) Review the PR, making comments and suggesting changes as appropriate.
+        When you have finished, approve the PR or request changes, based on your review.
+5. \(S) Respond to any comments from the Reviewer and/or, if any changes were requested,
+        make those and re-request a review when you have finished.
+6. \(R) When all changes have been made to your satisfaction,
+        approve the PR, thank the Submitter for their contribution,
+        and merge the PR
+        (Or close it if the changes were made for practice but should not
+        be included in the actual lesson.)
+
+::::::::::::::::
+
+
+:::::: callout
+
+## Effective review comments are a learned skill
```

Comment:
```suggestion
## Reviewing is a Skill
```

A shorter title, that has the advantage of following a theme from Instructor Training.